dockerurl/dependencies.go

```diff
@@ -11,12 +12,24 @@ import (
 )
 
 func (u *Updater) Dependencies(_ context.Context) ([]updater.Dependency, error) {
-	return docker.ExtractDockerfileDependencies(u.root, u.extractDockerfile)
+	var deps []updater.Dependency
+	err := docker.WalkDockerfiles(u.root, u.pathFilter, func(path string, parsed *parser.Result) error {
+		fileDeps, err := extractDockerfile(parsed)
+		if err != nil {
+			return fmt.Errorf("extracting dependencies: %w", err)
+		}
+		deps = append(deps, fileDeps...)
+		return nil
+	})
+	if err != nil {
+		return nil, fmt.Errorf("collecing dependencies: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("collecting dependencies: %w", err)
```

Guess when I copy-pasted this (before noticing this in the other PR).